modules/service/src/main/scala/lucuma/odb/data/CallForProposalsType.scala

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) 2016-2023 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package lucuma.odb.data
+
+import lucuma.core.util.Enumerated
+
+enum CallForProposalsType(val tag: String) {
```

Comment:
It seems we'll be wanting these enums available in Explore. We have been moving them to `core`, but another possibility would be to move them to the odb schemas module?

Reply:
I was planning on putting it in `core` since that's where everything else goes.  Putting it in `odb-schemas` would be easier though if satisfactory.  All schemas has at the moment are codecs and the `OdbSchema.graphql` itself.

Reply:
Just leave it for now. Since the enum doesn't contain any information than the tag, we can start by using the enum generated by clue.